packages/amplify-cli/src/utils/team-provider-migrate.ts

```diff
@@ -0,0 +1,22 @@
+import { Context } from '../domain/context';
+import { stateManager, pathManager } from 'amplify-cli-core';
+import _ from 'lodash';
+import { externalAuthEnable } from 'amplify-category-auth';
+const message = `Amplify auth will be modified to mangage secrets from deployment-secrets.json. Would you like to proceed?`;
+
+export async function MigrateTeamProvider(context: Context): Promise<boolean> {
```

Comment:
function name should start with lowercase. Also use arrow notation. Also would be nice to have a short comment on what the return value means (looks like it just returns whether or not the migration logic was executed)